guide.adoc

```diff
@@ -181,6 +181,10 @@ include::{chapters}primitive_topology.adoc[]
 
 include::{chapters}mapping_data_to_shaders.adoc[]
```

Comment:
bonus would be to link the Storage Image and Texel Buffer references from this chapter to your new chapter